src/string-logger.js

```diff
@@ -18,12 +20,17 @@ class StringLoggerInternal implements IStandardLog {
     _groupIndent: number = 0;
 
     _log = (...args: Array<string>) => {
-        const sep = getPathSeparator();
+        const {sep} = path;
         /**
          * We want to normalize the string in case it contains filepaths.
          * This ensures that snapshots are standardized across platforms.
+         *
+         * We do two things:
+         * - Replace the root path so that it's platform-independent
+         * - Replace the path separator so that it's platform-independent
          */
-        const regex =
+        const rootDirRegex = new RegExp(escapeRegExp(ROOT_DIR), "g");
```

Comment:
The verbose logging mode includes file paths - to make sure they pass, we strip the root dir off them.